docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,155 @@
+
+## AbstractJdbcPollInputOperator
+This is the abstract implementation that serves as base class for polling messages from jdbc store. It can be extended to modify functionality or add new capabilities. This class doesn’t have any ports, so concrete subclasses will need to provide them if necessary.
+
+![AbstractJdbcPollInputOperator.png](images/jdbcinput/operatorsClassDiagram.png)
+
+###<a name="AbstractJdbcPollInputOperatorProps"></a>Properties of AbstractJdbcPollInputOperator
+Several properties are available to configure the behavior of this operator and they are summarized in the table below. Of those only store, tableName, columnsExpression and key are mandatory ones. Those properties can be set like this:
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.prop.tableName</name>
+  <value>mytable</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.columnsExpression</name>
+  <value>column1,column2,column4</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.key</name>
+  <value>keycolumn</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.databaseDriver</name>
+  <value>com.mysql.jdbc.Driver</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.databaseUrl</name>
+  <value>jdbc:mysql://localhost:3306/mydb</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.connectionProps</name>
+  <value>user:myuser,password:mypassword</value>
+</property>
+```
+
+* If you need to filter the table records, set `whereCondition` which will be added to the generated sql query.
+* If you have huge database to scan set `partitionsCount` to a higher number to increase read parallelism.
+* The operator uses PreparedStatement, a precompiled SQL statement to fetch records from database table. You can set `fetchSize` to hint database driver to restrict number of rows to fetch in one call. The remaining rows of resultset will be fetched in subsequent calls.
+* The platform invokes the emitTuples() callback multiple time in each streaming window; within a single such call, if a large number of tuples are emitted, there is some risk that they may overwhelm the downstream operators especially if they are performing some compute intensive operation. For such cases, output can be throttled by reducing the value of the `batchSize` property. Conversely, if the downstream operators can handle the load, increase the value to enhance throughput.
+* If there is high frequency of incoming records in your table and you want to process it as soon they appear use lower value of `pollInterval`; if they appear rarely or if some delay in processing a new file is acceptable, increase it.
+* After reading the records from table they are held in memory for some time till they are emitted to next operator. The records are kept in a blocking queue. The capacity of this blocking queue can be changed using parameter `queueCapacity`.
```

Comment:
How does the reader decide when to change it ?
"from table" => "from the table"
The reason for being "held in memory" should be clarified by describing the asynchronous nature of the fetching
by a worker thread and the limits on how much data the operator thread can dequeue and emit in one call.